src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/newline.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'newline'
+})
+export class NewLinePipe implements PipeTransform {
```

Comment:
It seems that the pipe behavior is to transform an array to HTML text (each item in a new line),
so wouldn't be better to name it "ListToHtmlPipe" or something similar?

Please provide the corresponding .spec.ts file.

Reply:
Thanks @alfonsomthd . Renamed and added the test file.